package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "lint": "eslint .",
-    "tap": "tap test/test.js",
+    "tap": "tap test",
```

Comment:
Renaming the file does not need to be part of this change.

Reply:
I agree that it doesn't need to be part of this change, but I also think that it would be nice to have a better naming pattern (e.g. src.js -> src_test.js). This will make it easier to test different files in the future. What do you think?

Reply:
Let's leave it out of this PR. There's won't be different files in the future, considering the focus of this package, so I'm fine with just leaving things are they are.